.github/workflows/test.yaml

```diff
@@ -28,7 +28,9 @@ jobs:
         run: |
           nix develop -c ./fixture/package.sh
       - name: Run tests
-        run: cargo test --no-fail-fast
+        env:
+          RUST_LOG: warn,holochain_http_gateway=info
+        run: cargo test --no-fail-fast -- --test-threads=1
```

Comment:
Sweettests are so heavy, running them in parallel doesn't help. We could be a bit more selective if we used nexttest here and defined the websocket tests as needing to run in sequence but... bigger job than I want to take on this in PR.